docs/redis/redis.md

```diff
@@ -362,3 +362,11 @@ we split the key set into 5 different Scylla table. The shared commands
 should operate on these tables in parallel. For instance, DEL command,
 which allows user to remove any key in Redis database, should delete the
 given key from 5 different Scylla tables.
+
+### 5.7 Atomicity Guarantee
+
+* "mset" implementation offers no atomicity guarantee.
+
+https://redis.io/commands/mset explains that "all given keys are set at once.
+It is not possible for clients to see that some of the keys were updated while
+others are unchanged." - but this is not promised by this implementation.
```

Comment:
It should also be noted that Scylla cannot guarantee that the write will succeed, like redis' implementation does.

Reply:
Yes, we should have such comment, but it should be much bigger than "mset": You should say something like the following:

1. Many of Redis's operations (including set, mset, etc.) promise in the documentation that they cannot fail, but in Scylla such writes may fail in rare cases of errors: Although Scylla is highly-available by design, it isn't guaranteed always-available.
2. However, this isn't actually a Scylla-specific problem: Despite what the Redis documentation promises, even in the original Redis every request - including the fail-proof promise in the documentation can actually fail. So this promise is ncorrect! As pointed out, for example, in https://stackoverflow.com/questions/12911248/redis-set-command-cant-fail-but-can: Even in the regular Redis, certain problems - specifically running out of memory - can cause operations to fail, even set or mset. So Scylla's failures in this cases will not introduce a new problem for clients.